src/js/components/DSLFormDropdownPanel.js

```diff
@@ -56,6 +56,7 @@ class DSLFormDropdownPanel extends React.Component {
    */
   handleChange(expression) {
     this.setState({ expression });
+    this.props.onChange(expression);
```

Comment:
Wonder if we should we go with 
```js
this.setState({ expression }, () => this.props.onChange(expression));
```

Reply:
This is the exact same code we have in master that fixed it, do you want me to change it in both or in none of them?

Reply:
I expected this to be a cherry-pick, yes. I just wonder if it's ok to have those values out of sync because `setState` is an async function. Sometimes it leads to interesting bugs. So I'm just asking whether we should change it or we totally sure values being out of sync is not an issue.

Reply:
I haven't experienced any problems in that regard, but of course you can never be 100% sure. I can fix it and update it on master, too. This time I am going to create a new subticket